### PR TITLE
Debug log incoming lines

### PR DESCRIPTION
**Pull request by l15k4**
For better trouble shooting, see https://github.com/prometheus/graphite_exporter/issues/57

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /.build
 /.release
 /.tarballs
+.idea/
```
Comment (matthiasr) [inline .gitignore]: I'd rather not have editor-specific settings in here. You can ignore .idea for yourself in `.git/info/exclude` (this repository only) or [create a global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for yourself.

**Comment by l15k4** [comment]
@matthiasr what do you mean by `add the sign-off to the actual change` ? I've never used golang so I'm having troubles with outputting the final sample, I don't know how to print it properly. Would you mind supplying that for yourself ?

**Comment by matthiasr** [comment]
For the sign-off please look at the details of the failing DCO check.

**Comment by matthiasr** [comment]
Since you need to rebase for the sign-off anyway, please drop the two commits relating to gitignore. You can do that with `git rebase -i`.

**Comment by matthiasr** [comment]
Thank you!

**Comment by l15k4** [comment]
@matthiasr sorry for bothering again, but should the debug logging be enabled by adding `-log.level=debug` flag ? I can't make it work, this is my first encounter with golang

**Comment by l15k4** [comment]
Can pls anybody help how to enable the debug logging ?

**Comment by matthiasr** [comment]
try `--log.level=debug` (two dashes)

**Comment by l15k4** [comment]
I tried both, the repro is simple : 
`docker-compose.yml`
```
version: "2"

services:
  graphite-exporter:
    image: prom/graphite-exporter
    ports:
      - "9108:9108"
      - "9109:9109"
      - "9109:9109/udp"
    volumes:
      - ${PWD}/graphite_mapping.conf:/tmp/graphite_mapping.conf
    command: --graphite.mapping-config=/tmp/graphite_mapping.conf --log.level=debug
```
Then  `docker exec -it graphiteexporter_graphite-exporter_1 sh`
```
echo "druid.druid/historical.jvm/gc/cpu 56867252 1546008094" | nc -u -w1 localhost 9109
echo "druid.druid/historical.jvm/gc/cpu 56867252 1546008094" | nc localhost 9109
```

Nothing happens ... what am I missing here, these are the logs : 
```
graphite-exporter_1  | time="2018-12-28T15:50:46Z" level=info msg="Starting graphite_exporter (version=0.3.0, branch=HEAD, revision=4b03941683bd3984b5023ea3fef8e38796cbd2ee)" source="main.go:217" 
graphite-exporter_1  | time="2018-12-28T15:50:46Z" level=info msg="Build context (go=go1.10.3, user=root@2fc574c39d59, date=20180822-07:47:24)" source="main.go:218" 
graphite-exporter_1  | time="2018-12-28T15:50:46Z" level=info msg="Listening on :9108" source="main.go:282" 
```

**Comment by l15k4** [comment]
omg, my fault, it works if I change to `image: prom/graphite-exporter:v0.4.2` ... Sorry

**Comment by matthiasr** [comment]
maybe you need to `docker pull prom/graphite-exporter`?

**Pull request merged**